pyds8k/resources/ds8k/v1/common/mixins.py

```diff
@@ -436,14 +437,42 @@ def get_flashcopies(self, fcid=None):
 
     def get_flashcopy(self, fcid=None):
```

Comment:
Please don't modify get_flashcopy. It may effect other projects. Please add new function get_cs_flashcopies for cs/flashcopies.

Reply:
done